packages/maps/src/components/result/ReactiveGoogleMap.js

```diff
@@ -354,17 +354,17 @@ class ReactiveGoogleMap extends Component {
 }
 
 ReactiveGoogleMap.propTypes = {
-	addComponent: types.funcRequired,
-	setMapData: types.funcRequired,
-	loadMore: types.funcRequired,
-	removeComponent: types.funcRequired,
-	setQueryListener: types.funcRequired,
+	addComponent: types.func,
+	setMapData: types.func,
+	loadMore: types.func,
+	removeComponent: types.func,
+	setQueryListener: types.func,
```

Comment:
I have one question, what's the need to define these props here.
Since we're not using it here, we can simply remove it.

Reply:
True, let me just add props that are part of api